godtools/App/Features/AppLanguage/Presentation/DownloadableLanguages/DownloadableLanguagesViewModel.swift

```diff
@@ -77,30 +75,27 @@ extension DownloadableLanguagesViewModel {
     
     func downloadableLanguageTapped(downloadableLanguage: DownloadableLanguageListItemDomainModel) {
         
-        // TODO: - This is for UI demo purposes only.  Remove once language download functionality is implemented.
-                
         switch downloadableLanguage.downloadStatus {
+            
         case .notDownloaded:
             
-            updateDummyLanguage(with: .downloading(progress: 0))
+            downloadToolLanguageUseCase.downloadToolLanguage(downloadableLanguage.languageId)
+                .sink { _ in
+                    
+                }
+                .store(in: &cancellables)
```

Comment:
The one caveat to storing a cancellable here is if the View is existed while a download process is happening, that download would be cancelled, which may not be what we want.  I'd confirm with Jill if that is the behavior they want.

One solution I found to keep publishers alive is to have a static cancellable that references the result of the publisher chain.  Not sure if there's a cleaner or preferred way, would be curious to investigate that more.

Reply:
Actually static variable wouldn't work here because multiple languages could be downloading.

Reply:
Confirmed with Jill that the download should complete even if the page is closed.  I tried to do some googling on retaining subscriptions until completion, but didn't find much.  Only alternative I saw was to create a custom publisher that makes sure deallocation doesn't occur until the task is complete.  Seems easier to just keep it static, but what do you think?  For now, I just made a static cancellables set.

Reply:
Do you the link for the custom publisher?  I'm curious to see what that entails.

And for the static array I'm curious to see if those cancellable's get removed when the publisher completes.  Let's leave the static array.  I'm curious to test that some more when GT-2234 is completed.

Reply:
Taking a second look, [this](https://forums.swift.org/t/combine-best-practices-with-memory-management-using-subjects-to-publish-values/38724/2) probably wouldn't solve our problem.  Although it is interesting, and possibly prone to memory leaks if you're not careful.  But yeah, seems like even if we did a custom class of some sort, we'd still end up needing something `static` to keep it alive.

Reply:
Thanks @rachaelblue, yeah I like the static approach better, curious to test that some more though to see if those cancellable's are removed from the static array reference.  That's my only concern is leaving those in memory.

Reply:
The static references for those single cancellable's I should check too.